.travis.yml

```diff
@@ -14,6 +14,8 @@ matrix:
   - go: tip
 
 before_install:
+- go get -u github.com/golang/dep/cmd/dep
+- dep ensure
```

Comment:
You want to use `dep ensure -vendor-only` to not run an update before fetching vendors.